classes/jplain.lua

```diff
@@ -9,7 +9,6 @@ class._name = "jplain"
 
 function class:_init (options)
    tplain._init(self, options)
-   SILE.languageSupport.loadLanguage("ja")
    SILE.settings:set("document.language", "ja", true)
```

Comment:
Just a question here. Should we rather always use `SILE.call("language"..."` rather than using the low-level setting? The difference is not that clear. Formally we use a bunch of things indeed (languageSupport, setting), but why not the high level command directly?

Reply:
(While at it, same remark for the next line, what behavior do we expect being different from setting the font or using a call to `\font`?  That's also a matter of showing what we expect developers to use in their own classes/packages, i.e. which paradigms are expected)

Reply:
The difference is not clear because it's an absolute scramble. At the moment they are both effectively equivalent. We'll have a clearer answer to this after we sort out the whole language support module layout. For now about the only comment I have is that manually loading the language support module before using it in a setting or font/language command call should *not* be necessary any more.